socorro/unittest/signature/test_rules.py

```diff
@@ -1281,58 +1281,73 @@ def test_action_success_long_message(self):
         expected_sig = 'Abort | {}... | hello'.format('a' * 77)
         assert processed_crash['signature'] == expected_sig
 
-    def test_action_success_remove_unwanted_parts(self):
-        rule = AbortSignature()
-
-        raw_crash = {}
-        processed_crash = {}
-
-        # Test with just the "ABOR" thing at the start.
-        processed_crash['signature'] = 'hello'
-        raw_crash['AbortMessage'] = '[5392] ###!!! ABORT: foo bar line 42'
-
-        action_result = rule.action(raw_crash, processed_crash, [])
-
-        assert action_result is True
-        assert processed_crash['original_signature'] == 'hello'
-        assert processed_crash['signature'] == 'Abort | foo bar line 42 | hello'
-
-        # Test with a file name and line number.
-        raw_crash['AbortMessage'] = (
-            '[7616] ###!!! ABORT: unsafe destruction: file '
-            'c:/builds/moz2_slave/m-rel-w32-00000000000000000000/build/src/'
-            'dom/plugins/ipc/PluginModuleParent.cpp, line 777'
-        )
-        processed_crash['signature'] = 'hello'
+    @pytest.mark.parametrize('abort_msg, expected', [
```

Comment:
I redid this test because it's a bunch of cases with the same structure.

One thing about this test is that the input data in some cases involves very very long strings. I did this to keep the existing string. We could probably shorten the strings to smaller versions, though.

Reply:
no preference here